packages/core/test/ServerTest.ts

```diff
@@ -90,6 +95,33 @@ class WoTServerTest {
         expect(thing).to.have.property("properties").to.have.property("myProp");
     }
 
+    @test async "validate thing description"() {
```

Comment:
to be removed, or better to add a specific test for validation (e.g. in the helpers class)